lib/rucio/tests/test_replica_sorting.py

```diff
@@ -155,11 +156,11 @@ def test_replica_sorting(self):
                                    schemes=['root', 'gsiftp', 'davs'],
                                    metalink=True,
                                    client_location={'site': 'APERTURE'})
-        assert_in('domain="lan" priority="1">root://root.aperture.com:1409//test/chamber/mock/58/b5/element_0', ml)
-        assert_in('domain="lan" priority="2">davs://davs.aperture.com:443/test/chamber/mock/58/b5/element_0', ml)
-        assert_in('domain="wan" priority="3">gsiftp://gsiftp.blackmesa.com:8446/lambda/complex/mock/58/b5/element_0', ml)
-        assert_in('domain="wan" priority="4">davs://davs.blackmesa.com:443/lambda/complex/mock/58/b5/element_0', ml)
-        assert_in('domain="wan" priority="5">root://root.blackmesa.com:1409//lambda/complex/mock/58/b5/element_0', ml)
+        assert_in('domain="lan" priority="1" client_extract="False">root://root.aperture.com:1409//test/chamber/mock/58/b5/element_0', ml)
```

Comment:
XML convention is to have "true"/"false" (not uppercase)

Reply:
Will change.